### PR TITLE
integration tests

### PR DESCRIPTION
**Pull request by docteurklein**
Signed-off-by: Klein Florian florian.klein@free.fr

### CODE REVIEW

File: tests/integration/it_works.sh [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+    -P \
+    --name ci-test \
+    -e LOG_LEVEL=DEBUG \
+    docteurklein/compose-ci
+
+commit=$(git log -1 --format=%H)
+port=$(docker port ci-test 80/tcp)
+port=${port##*:}
+
+sleep 2
```
Comment (roukmoute) [inline tests/integration/it_works.sh:23]: random number ?

Comment (docteurklein) [reply tests/integration/it_works.sh:23]: enough for the http server to start theoretically

Comment (roukmoute) [reply tests/integration/it_works.sh:23]: Is it not possible to loop until a check is good?

Comment (docteurklein) [reply tests/integration/it_works.sh:23]: it is

**Pull request merged**